docs/source/advanced.rst

```diff
@@ -258,43 +258,187 @@ of :py:class:`scikeras.wrappers.BaseWrappers`. Currently, they are:
+    clf = KerasClassifier(
+        ...
+        loss=BinaryCrossentropy(label_smoothing=0.1),
+    )
+
+Keras, and SciKeras by extension, support passing a single loss (which will be applied to all outputs)
+or a loss for each output by passing a list of losses or a dict of losses.
+
+Additionally, SciKeras supports routed parameters to each individual loss, or to all losses together.
+
+.. code:: python
+
+    from tensorflow.keras.losses import BinaryCrossentropy, CategoricalCrossentropy
+
+    clf = KerasClassifier(
+        ...
         loss=[BinaryCrossentropy, CategoricalCrossentropy],
         loss__from_logits=True,  # BinaryCrossentropy(from_logits=True) & CategoricalCrossentropy(from_logits=True)
         loss__label_smoothing=0.1,  # passed to each sub-item, i.e. `loss=[l(label_smoothing=0.1) for l in loss]`
         loss__1__label_smoothing=0.5,  # overrides the above, results in CategoricalCrossentropy(label_smoothing=0.5)
     )
+    # or
+    clf = KerasClassifier(
+        ...
+        loss={"out1": BinaryCrossentropy, "out2": CategoricalCrossentropy},
+        loss__from_logits=True,  # BinaryCrossentropy(from_logits=True) & CategoricalCrossentropy(from_logits=True)
+        loss__label_smoothing=0.1,  # passed to each sub-item, i.e. `loss={o: l(label_smoothing=0.1) for o, l in loss.items()}`
+        loss__out2__label_smoothing=0.5,  # overrides the above, results in CategoricalCrossentropy(label_smoothing=0.5)
+    )
+
+With this parameter routing in place, you can now use hyperparameter tuning on each loss' parameters.
 
-With this parameter rounting in place, you can now use hyperparameter tuning on each loss' ``label_smoothing`` parameter individually.
+Metrics
++++++++
 
-Example: callback with routed parameters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Metrics have similar semantics to losses, but multiple metrics per output are supported.
+Here are several support use cases:
 
 .. code:: python
 
-    import tensorflow as tf
-    from sklearn.datasets import make_classification
-    from scikeras.wrappers import KerasClassifier
+    from tensorflow.keras.metrics import BinaryAccuracy, AUC
+
+    clf = KerasClassifier(
+        ...
+        metrics=BinaryAccuracy,
+        metrics__threashold=0.65,
+    )
+    # or to apply multiple metrics to all outputs
+    clf = KerasClassifier(
+        ...
+        metrics=[BinaryAccuracy, AUC],  # both applied to all outputs
+        metrics__0__threashold=0.65,
+    )
+    # or for different metrics to each output
+    clf = KerasClassifier(
+        ...
+        metrics={"out1": BinaryAccuracy, "out2": [BinaryAccuracy, AUC]},
+        metrics__out1__threashold=0.65,
+        metrics__out2__0__threashold=0.65,
+    )
+    # assuming you ordered your outputs like [out1, out2], this is equivalent to
+    clf = KerasClassifier(
+        ...
+        metrics=[BinaryAccuracy, [BinaryAccuracy, AUC]],
+        metrics__0__threashold=0.65,
+        metrics__1__0__threashold=0.65,
```

Comment:
```suggestion
        metrics__1__0__threshold=0.65,  # AUC(threshold=0.65)
```

Reply:
It would actually be `metrics__1__0__threshold=0.65,  # BinaryAccuracy(threshold=0.65)`.

`metrics__1__0__threshold` translates to: `metrics[1][0](threashold=0.65)`